global_func.php

```diff
@@ -1278,3 +1278,109 @@ function userBox(int|string $target_id): string
 {
     return $target_id;
 }
+
+/**
+ * @param string|array $permissions
+ * @param bool $exit
+ * @param int|null $target_id
+ * @return bool|void
+ */
+function check_access(string|array $permissions, bool $exit = true, ?int $target_id = null)
```

Comment:
This function has two unpredictable stats. This creates a confusing understanding of the usage

`check_access('test')` - will just exit, which cannot be represented easily in the editor and will look like the rest of the code will just run.
`check_access('test', false)` - returns an assertion based on the required parameters.

I would promote dropping the `bool $exit` parameter and always have this return a bool, and handle the eventual operation in the usage for better understandable code.

Reply:
Resolved by [af2ec41760a69976afcc9f7fd41548392cb4e30e](https://github.com/davemacaulay/mccodesv2/commit/af2ec41760a69976afcc9f7fd41548392cb4e30e)